.gitignore

```diff
@@ -41,4 +41,7 @@ lib
 services/**/daemon/*
 .tmp
 
-config/client_secret.json
\ No newline at end of file
+config/app.json
```

Comment:
I wanted to keep this file ignored so I don't commit my config by mistake. All configs are now in `examples/` for reference.